examples/demo_runner.py

```diff
@@ -136,6 +136,11 @@ def init_physics_test_scene(self, num_objects):
         object_lib_size = self._sim.get_physics_object_library_size()
         object_init_grid_dim = (3, 1, 3)
         object_init_grid = {}
+        if object_lib_size == 0:
```

Comment:
`assert object_lib_size > 0, "!!!No objects loaded in library, aborting object instancing example!!!"`

Reply:
Thanks